ADAL/src/public/ADUserInformation.h

```diff
@@ -34,15 +34,21 @@
 }
 
 /*! Factory method to extract user information from the AAD id_token parameter.
- @param idToken The contents of the id_token parameter, as passed by the server. */
-+ (ADUserInformation *) userInformationWithIdToken:(NSString *)idToken
-                                             error:(ADAuthenticationError * __autoreleasing *)error;
+ @param idToken The contents of the id_token parameter, as passed by the server.
+ @param homeUserId Combination of uid & utid.
+ */
++ (ADUserInformation *)userInformationWithIdToken:(NSString *)idToken
```

Comment:
this is a public method, so we need to keep it as is. Please create an internal method to pass homeUserId, or add 2 methods. In any case userInformationWithIdToken: error: has to stay.

Reply:
done